@ecomplus/widget-fb-pixel/src/lib/watch-app-routes.js

```diff
@@ -28,6 +28,9 @@ export default (fbq, options) => {
     }
 
     const emitCheckout = (step, option) => {
+      if (window.localStorage.getItem('fbq.orderIdSent')) {
+        window.localStorage.removeItem('fbq.orderIdSent')
+      }
```

Comment:
Por que isso? Se for assim melhor nem salvar isso no localStorage uai 🙃

Reply:
Tava sendo salvo pra evitar duplicidade, aí quando tem o evento você tá querendo deletar o item no storage, então ele passou a ser adicionado atoa, não?
Não acho que precisa mexer nisso aqui também, acho que vai é piorar outra coisa, isso aqui tinha uma razão de ser. Continuo achando que o erro no código estava no timeout, como estava antes não fazia sentido e daria exatamente no erro que você mencionou.

Reply:
Não mano, o problema do localstorage é, quando a pessoa está na página de finalização e ela recarrega a página e ai estava reenviando. O problema disso ai é, a pessoa comprou, cancelou e vai comprar de novo, não deveria contabilizar mais um novo pedido? Ocorreu comigo por exemplo de ficar no meu localstorage por mais tempo, aba ficou aberta mais de um dia e quando fui fazer o teste, não disparou evento por isso. Logo, se tiver no carrinho, pode limpar esse item.

Reply:
O timeout coloquei lá na outra pr

Reply:
> O problema disso ai é, a pessoa comprou, cancelou e vai comprar de novo, não deveria contabilizar mais um novo pedido?

Deveria, mas por isso ele salva o ID que foi enviado e checa ele depois https://github.com/ecomplus/storefront/blob/a988326f5011f1d1d6a76ebf3a7baed44f887a94/%40ecomplus/widget-fb-pixel/src/lib/watch-app-routes.js#L52
Quer dizer, se é um pedido novo ele passa aí mesmo tendo algo em `localStorage.getItem('fbq.orderIdSent')` porque deveria ser um ID diferente.

> O timeout coloquei lá na outra pr

Nessa PR aqui mesmo né...